src/Authentication/Services/SystemRegisterService.cs

```diff
@@ -114,5 +127,25 @@ public async Task<bool> DoesResourceIdExists(List<Right> rights, CancellationTok
 
             return true;
         }
+
+        /// <summary>
+        /// Checks if the access packages are found in access package list
+        /// </summary>
+        /// <param name="accessPackages">the list of access packages required by the system</param>
+        /// <param name="cancellationToken">the cancellation token</param>
+        /// <returns></returns>
+        public async Task<bool> HasValidAccessPackages(List<AttributePair> accessPackages, CancellationToken cancellationToken)
+        {
+            foreach (AttributePair accessPackage in accessPackages)
+            {
+                ServiceResource resource = await _resourceRegistryClient.GetResource(accessPackage.Value);
```

Comment:
Litt usikker på denne koden. HVa er det vi prøverå sjekke her. Hvorfor sjekker vi mot resource?

Reply:
jeg håpte at vi kunne integrere før PR gikk gjennom men nå skal jeg kommentere for tida. Det må integereres mot accesspackage metada api fra AM